src/main/sh/mommy

```diff
@@ -358,6 +359,54 @@ done
 
 shift "$((OPTIND - 1))"
 
+# do the symlinking
+if [ -n "$opt_rename" ]; then
+    if [ $# -lt 1 ]; then
+        die "mommy expected a file path to rename"
+    fi
+    echo "mommy is locating her shell script"
+    findoldpath=0
+    oldpath="$(whereis mommy | sed 's/^mommy://')"
+    
+    if is_blank "$oldpath"; then
+        echo "Mommy's shell script is not in path, mommy's gonna search your whole disk now"
+        echo "Warning: Mommy's search could be extremely slow on hard drives"
+        findoldpath=1
+        mommyshellscript="$(find / -name mommy -type f 2>/dev/null)"
+        mommymandocs="$(find / -name mommy.1.gz -type f 2>/dev/null)"
+        # prevents an error on my system, may not be applicable everywhere
+        sleep 1
+    fi
+
+    if [ "$findoldpath" -eq 0 ]; then
```

Comment:
these three lines can be replaced by an `else`.  you can then also remove `findoldpath`